datumaro/plugins/coco_format/converter.py

```diff
@@ -451,6 +453,67 @@ def save_annotations(self, item):
 
             self.annotations.append(elem)
 
+class _StuffConverter(_InstancesConverter):
+    pass
+
+class _PanopticConverter(_TaskConverter):
+    def save_categories(self, dataset):
+        label_categories = dataset.categories().get(AnnotationType.label)
+        if label_categories is None:
+            return
+
+        for idx, cat in enumerate(label_categories.items):
+            self.categories.append({
+                'id': 1 + idx,
+                'name': cast(cat.name, str, ''),
+                'supercategory': cast(cat.parent, str, ''),
+            })
+
+    def save_annotations(self, item):
+        if not item.has_image:
+            return
+
+        ann_filename = item.id + CocoPath.PANOPTIC_EXT
+
+        def id2bgr(id_map):
+            id_map_copy = id_map.copy()
+            bgr_shape = tuple(list(id_map.shape) + [3])
+            bgr_map = np.zeros(bgr_shape, dtype=np.uint8)
+            for i in range(3):
+                bgr_map[..., 2-i] = id_map_copy % 256
+                id_map_copy //= 256
+            return bgr_map
+
+        segments_info = list()
+        masks = []
+        for ann in item.annotations:
+            if ann.type != AnnotationType.mask:
+                continue
+
+            segment_info = {}
+            segment_info['id'] = ann.id
```

Comment:
It seems that we need a fallback to generate these values, when they are missing (`None`).

Reply:
done